index.bs

```diff
@@ -182,9 +182,9 @@ The [=AV1 Image File Format=] defines multiple profiles, which restrict the allo
 The profiles defined in this specification follow the conventions of the [[!MIAF]] specification.
 Images encoded with AV1 and not meeting the restrictions of the defined profiles may still be compliant to this [=AV1 Image File Format=] if they adhere to the general AVIF requirements.
 
-[=AV1 Image File Format=] supports High Dynamic Range (HDR) and Wide Color Gamut (WCG) images as well as Standard Dynamic Range (SDR). It supports monochrome images as well as multi-channel images with all the bit depths and color spaces specified in [[!AV1]].
+The [=AV1 Image File Format=] supports High Dynamic Range (HDR) and Wide Color Gamut (WCG) images as well as Standard Dynamic Range (SDR). It supports monochrome images as well as multi-channel images with all the bit depths and color spaces specified in [[!AV1]], and beyond with [=Sample Transform Derived Image Items=]. The [=AV1 Image File Format=] also supports transparency/translucency through an auxiliary [=AV1 bitstream=].
```

Comment:
Nit: I suggest changing "transparency/translucency" to "transparency (alpha)".

Should we also mention depth maps?

Reply:
I agree with the suggestion. I guess we could mention that other types of auxiliary streams are also possible.

Reply:
Done.